tests/providers/opensearch/operators/test_opensearch.py

```diff
@@ -0,0 +1,126 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from opensearchpy import Document, Keyword, Text
+
+from airflow.models import DAG
+from airflow.providers.opensearch.operators.opensearch import (
+    OpenSearchAddDocumentOperator,
+    OpenSearchCreateIndexOperator,
+    OpenSearchQueryOperator,
+)
+from airflow.utils.timezone import datetime
+
+TEST_DAG_ID = "unit_tests"
+DEFAULT_DATE = datetime(2018, 1, 1)
+MOCK_SEARCH_RETURN = {"status": "test"}
+
+
+class FakeDocument(Document):
+    title = Text(fields={"raw": Keyword()})
+    author = Text()
+    published = Text()
+
+    def save(self, **kwargs):
+        return super().save(**kwargs)
+
+
+class TestOpenSearchQueryOperator:
+    def setup_method(self):
+        args = {
+            "owner": "airflow",
+            "start_date": DEFAULT_DATE,
+        }
+
+        self.dag = DAG(f"{TEST_DAG_ID}test_schedule_dag_once", default_args=args, schedule="@once")
```

Comment:
Nitpick, and doesn't REALLY matter since it's a unit test, but this would parse out to "unit_teststest_schedule_dag_once", note the second "word" gets smooshed.

Reply:
Fixed in latest push.